easylist/easylist_general_block.txt

```diff
@@ -3375,6 +3375,7 @@
 /ame-dfp/*
 /amp-ad-
 /amp-auto-ads-
+/amp-sticky-ad-
```

Comment:
This is not needed by another rule `-sticky-ad-`.

Reply:
```suggestion
```